ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py

```diff
@@ -548,3 +549,35 @@ def test_jax_numpy_full(
         fill_value=fill_value,
         dtype=dtype,
     )
+
+
+def test_jax_numpy_double(
+    *,
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    # Check if the input datatype is double
+    if dtype_and_x[0] != "float64":
```

Comment:
This condition should be removed because we should never `return` from a test, to skip certain cases hypothesis provides the `assume` function, or you can use the `filter` method on strategies, if any of this does not make sense, I suggest reading the testing docs or hypothesis docs, regardless, we do not want to skip float64 cases as they are valid cases.